### PR TITLE
modify English document while_loop

### PR DESCRIPTION
**Pull request by gfwm2013**
如题，修复while_loop英文文档
192 168 1 102_8088_documentation_docs_en_api_layers_while_loop html (1)

### CODE REVIEW

File: python/paddle/fluid/layers/tensor.py [hunk 1]
```diff
@@ -448,13 +448,13 @@ def assign(input, output=None):
     The OP copies the :attr:`input` to the :attr:`output`.
 
     Parameters:
-        input (Variable|numpy.ndarray): A tensor or numpy ndarray, its data type supports
+        input (Variable|numpy.ndarray): A tensor, numpy ndarray or LoDTensorArray, its data type supports
```
Comment (zhhsplendid) [inline python/paddle/fluid/layers/tensor.py]: Why did you change the English doc of assign Op but not Chinese doc?

Comment (zhhsplendid) [inline python/paddle/fluid/layers/tensor.py]: A better order will be "A tensor, or LoDTensorArray, or numpy ndarray"

File: python/paddle/fluid/layers/tensor.py [hunk 2]
```diff
@@ -466,6 +466,14 @@ def assign(input, output=None):
           fluid.layers.assign(data, result1) # result1 = [[2.5, 2.5], [2.5, 2.5], [2.5, 2.5]]
           result2 = fluid.layers.assign(data)  # result2 = [[2.5, 2.5], [2.5, 2.5], [2.5, 2.5]]
           result3 = fluid.layers.assign(np.array([[2.5, 2.5], [2.5, 2.5], [2.5, 2.5]], dtype='float32')) # result3 = [[2.5, 2.5], [2.5, 2.5], [2.5, 2.5]]
+
+          i = fluid.layers.fill_constant(shape=[1], value=0, dtype='int64')
+          array = fluid.layers.array_write(x=data, i=i)
+          result_array1 = fluid.layers.create_array(dtype='float64')
+          fluid.layers.assign(array, result_array1)
+          result_array2 = fluid.layers.assign(array)
+          result4 = fluid.layers.array_read(array=result_array1, i=i) # result4 = [[2.5, 2.5], [2.5, 2.5], [2.5, 2.5]]
```
Comment (zhhsplendid) [inline python/paddle/fluid/layers/tensor.py]: result4, 5 is very ugly. (result 1, 2 above is fine since number is small).
Let's change "result4, 5" into "array_elem1, 2".

Comment (gfwm2013) [reply python/paddle/fluid/layers/tensor.py]: Done.

File: python/paddle/fluid/layers/tensor.py (hunk 1 above)
Comment (gfwm2013) [reply python/paddle/fluid/layers/tensor.py]: Done.

Comment (zhupengyang) [reply python/paddle/fluid/layers/tensor.py]: - Tensor and LoDTensorArray are belong to Variable. So, "A Tensor, or LoDTensorArray, or numpy ndarray" is better
- tensor -> Tensor

**Review by saxon-zh — APPROVED** [review]

**Review by zhhsplendid — APPROVED** [review]
LGTM
